.github/workflows/on-push-or-pr-to-main.yml

```diff
@@ -26,6 +26,17 @@ jobs:
           rubocop_extensions: rubocop-performance:gemfile rubocop-rake:gemfile rubocop-rspec:gemfile
           reporter: github-pr-review
 
+  readme:
+    steps:
+      - name: Generate README
+        uses: momentohq/standards-and-practices/github-actions/generate-and-commit-oss-readme@gh-actions-v1
+        with:
+          project_status: official
+          project_stability: alpha
+          project_type: sdk
+          sdk_language: Ruby
+          usage_example_path: ./examples/basic.rb
```

Comment:
this part looks like what i'd expect, but there's a bunch of other stuff below this that i wouldn't expect in this PR?  (other than the README template obvs).  Wondering if this PR was intended to target the previous PR as a parent?

Reply:
This was not ready for review; I noted that in the description but it was an edit so maybe that didn't come across.

This was on top of the YARD doc branch. I wanted to test the action, so I used main as the parent to test the action. Now that the YARD doc branch is merged it's all good (though the action is busted)

Reply:
sorry, i did see that comment but I presumed that when you added me as a reviewer later, that meant that it was ready

Reply:
My bad.